src/styles/index.css

```diff
@@ -0,0 +1,4 @@
+@import url(./blocks/card.css);
+@import url(./blocks/stars.css);
+
+@import url("https://fonts.googleapis.com/css2?family=Roboto&display=swap");
```

Comment:
It's solution doesn't work, becouse U've mislinked your fonts, and it's a reason why your test doesn't correct).